tests/test_utils/test_transactiontestcase.py

```diff
@@ -26,3 +46,22 @@ def test(self, call_command):
             reset_sequences=False, inhibit_post_migrate=True,
             database='default', verbosity=0,
         )
+
+
+class TestDataConsistencyOnTearDown(TestSerializedContentMockMixin, TransactionTestCase):
+    """
+    TransactionTestCase._fixture_teardown() is flushing the whole database.
+    If an other test after this one needs some data coming from initial data migrations,
+    there data won't be found.
+    One solution is to load again initial data migrations after the flush called on _fixture_teardown().
+    """
+    available_apps = ['test_utils']
+    serialized_rollback = True
+    initial_data_migration = '[{"model": "test_utils.car", "pk": 1, "fields": {"name": "K 2000"}}]'
+
+    def _post_teardown(self):
```

Comment:
In this example yes. But my PR is updating some code that is related to a post-test cleaning, not a post-class cleaning, so I test my post-test cleaning.

In this example, I agree that the 2 of them are close. By testing with `_post_teardown`, I also ensure that all the tests, if people are using `TransactionTestCase` with several tests, will have the proper data. By testing with `tearDownClass`, I just test that my database is in a correct state after a whole suite of tests included in a `TransactionTestCase`.

This is a disturbing because I am using a `TransactionTestCase` to test a functionality of the inner logic of `TransactionTestCase`.